contrib/tests/actions/chains/test_packs_pack.yaml

```diff
@@ -120,7 +120,7 @@ chain:
         params:
             env:
               ST2_AUTH_TOKEN: "{{token}}"
-            cmd: "st2 run packs.load register=all | grep -c 'Registered' | grep 3"
+            cmd: "st2 run packs.load register=all | grep -c 'Registered' | grep 4"
```

Comment:
This `4` could be an input variable defined in the workflow. So we run the validate workflow with this number passed in.